packages/graphql-serve/README.md

```diff
@@ -76,45 +77,24 @@ $ gqlserve --help
 Usage: gqlserve [options]
```

Comment:
```suggestion
Usage: gqlserve serve [options]
```

Reply:
`gqlserve` on its own makes more sense than `gqlserve serve`. But currently the latter is what we have. Might it make more sense to remove the `serve` subcommand?

Reply:
I would do that, but currently the `graphback-cli` also uses the commands  i.e. it also has the serve and print-schema commands, let me look into it